tpkg/ixfr_faildel.tdir/ixfr_faildel.test

```diff
@@ -7,8 +7,6 @@
 PRE="../../.."
 DIG=dig
 
-sleep 6
```

Comment:
Maybe best to keep this for slow executing test hosts.

Reply:
Fixed.